.eslintrc.js

```diff
@@ -1,18 +1,19 @@
 "use strict";
 
+// eslint-disable-next-line no-undef -- Keep backward compatibility with CommonJS.
 module.exports = {
   parserOptions: {
     ecmaVersion: 2019,
+    sourceType: "module",
   },
   env: {
     es6: true,
     node: true,
   },
-  plugins: ["eslint-comments", "jest", "node", "sort-requires"],
```

Comment:
[note] This config is needless because they are automatically set via `extends`:

- `plugin:eslint-comments/recommended`
- `plugin:jest/recommended`
- `plugin:node/recommended-module`